lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -78,6 +84,72 @@ func (i *InteractionHandlersImpl) Configure(api *operations.PortLayerAPI, _ *Han
 	}
 }
 
+func (i *InteractionHandlersImpl) JoinHandler(params interaction.JoinParams) middleware.Responder {
+	defer trace.End(trace.Begin(params.ID))
+
+	handle := exec.GetHandle(params.ID)
```

Comment:
this should be params.Config.Handle (see ScopesAddContainer, but ignore the naming) - think of the case when the port layer components are not co-located... the handle in that scenario is the full textual representation of the config spec, which this call will modify.